paddle/fluid/operators/distributed/grpc_variable_response.cc

```diff
@@ -293,6 +293,14 @@ int GRPCVariableResponse::Parse(Source* source) {
         }
         break;
       }
+      case sendrecv::VariableMessage::kTrainerIdFieldNumber: {
```

Comment:
Maybe it's more convenient to add another PR to add trainer_id and pserver can enforce a message from a trainer wouldn't duplicate.